articles/virtual-desktop/virtual-desktop-fall-2019/set-up-scaling-script.md

```diff
@@ -39,7 +39,7 @@ During peak usage time, the job checks the current number of sessions and the VM
 During the off-peak usage time, the job determines how many session host VMs should be shut down based on the *MinimumNumberOfRDSH* parameter. If you set the *LimitSecondsToForceLogOffUser* parameter to a non-zero positive value, the job will set the session host VMs to drain mode to prevent new sessions from connecting to the hosts. It will then notify any currently signed in users to save their work, wait the configured amount of time, and then force the users to sign out. Once all user sessions on the session host VM have been signed out, the job will shut down the VM. After the VM is shut down, the job will reset its session host drain mode.
 
 >[!NOTE]
->If you manually set the session host VM to drain mode, the session host VM will not be managed by the job. If the session host VM is running and set to drain mode, it will be treated as unavailable and so the job may start additional VMs to handle the load.
+>If you manually set the session host VM to drain mode, the session host VM will not be managed by the job. If the session host VM is running and set to drain mode, it will be treated as unavailable and so the job may start additional VMs to handle the load. However, in order to prevent the job from managing any session host VMs that need some maintenance, you can set a Tag to those Azure VMs and specify its name as a value to *MaintenanceTagName* parameter when you create Azure Logic App Scheduler later.
```

Comment:
Our recommendation to take session host VM's out for any kind of maintenance would be to use a a Tag to those Azure VMs before you set it to drain mode. You specify tag name as a value to *MaintenanceTagName* parameter when you create Azure Logic App Scheduler later. This will help you distinguish these VM's from one's being managed by scaling service. Setting the maintenance tag prevents the VM from being managed by scaling service till the tag is removed.